schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -88,53 +92,496 @@ protocol CommonInterpreted {
         inrepeat
     }
 
+
     /**
-    This object holds all the information related to a specific variant observation in a given sample, including
-    zygosity, phase, depth of coverage, variant allele frequency and allele origins.
+    Variant tiers as defined by Genomics England
     */
-    record VariantCall {
+    enum Tier {NONE, TIER1, TIER2, TIER3, TIER4, TIER5}
+
+    /**
+    The reference genome assembly
+    */
+    enum Assembly {GRCh38, GRCh37}
+
+    enum ClinicalSignificance {
+        benign,
+        likely_benign,
+        likely_pathogenic,
+        pathogenic,
+        uncertain_significance
+    }
+
+    enum DrugResponseClassification {
+        altered_sensitivity,
+        reduced_sensitivity,
+        increased_sensitivity,
+        altered_resistant,
+        increased_resistance,
+        reduce_resistance,
+        increased_toxicity,
+        reduced_toxicity,
+        adverse_drug_reaction,
+        indication,
+        contraindication,
+        dosing_alteration,
+        increased_dosing,
+        reduced_dosing,
+        increased_monitoring,
+        increased_efficacy,
+        reduced_efficacy
+    }
 
+    enum PrognosisClassification{
+        altered_prognosis,
+        favorable_prognosis,
+        unfavorable_prognosis
+    }
+
+    enum TraitAssociation {
```

Comment:
Many types from the evidence entry model. I know it's probably a pain but should reference them rather than making another copy here - will probably end up badly otherwise?